glide.yaml

```diff
@@ -12,3 +12,7 @@ import:
   version: master
 - package: github.com/giantswarm/operatorkit
   version: master
+- package: github.com/prometheus/prometheus
+  version: v1.6.3
+- package: github.com/prometheus/common
```

Comment:
`prometheus/common` is a heap of utils code, I have no issue with `master` here - additionally, they don't specify releases anyway.